## /exceptions.util_exceptions.py
import logging
logger = logging.getLogger(__name__)

